### DISCUSSION TITLE
No data on JSON type

### DISCUSSION THREAD

**Discussion by jimayyyy** [grafana/grafana-infinity-datasource#574] [Q&A]
Hi, i'm actually using grafana infinity, i don't know why, 
my query show result on UQL but not on JSON type.
![image](https://user-images.githubusercontent.com/72018578/230619382-6eb30eeb-ce49-4f87-9dd5-7a80ba941876.png)
![image](https://user-images.githubusercontent.com/72018578/230619548-77035fff-1d9f-4d1d-9924-30cd1ae4ea82.png)
I don't understand why... can someone help me fix this issue please ?

**Comment by yesoreyeram** [comment]
Can you provide more details? Steps to reproduce,  What your response looks like? What your queries looks like etc.. half screenshot of the queires doesn't provide any values for investigation.

**Reply by yesoreyeram** [reply]
Also use the latest version of the plugin and seems pretty outdated
